run/docker-compose.yml

```
@@ -1,4 +1,4 @@
-version: "3.8"
+version: "3.9"
```

If we upgrade the docker-compose file version, we might force people to upgrade theri local version of docker-compose. This is probably not necessary here?

```suggestion
version: "3.8"
```